util/constants.js

```diff
@@ -29,3 +31,23 @@ export const ONE_IN_WEI = '1000000000000000000';
 export const HUNDRED_K_OLAS_IN_WEI = '100000000000000000000000';
 
 export const VEOLAS_URL = 'https://govern.olas.network/veolas';
+export const GOVERN_APP_URL = 'https://govern.olas.network';
+export const OPERATE_APP_URL = 'https://operate.olas.network';
+
+export const STAKING_CONTRACTS_DETAILS = {
+  '0x00000000000000000000000095146adf659f455f300d7521b3b62a3b6c4aba1f': {
+    name: 'Contribute Alpha',
+    totalBond: 100,
+    tweetsPerEpoch: 1,
+  },
+  '0x0000000000000000000000002c8a5ac7b431ce04a037747519ba475884bce2fb': {
+    name: 'Contribute Alpha 2',
+    totalBond: 100,
+    tweetsPerEpoch: 1,
+  },
+  '0x000000000000000000000000708e511d5fcb3bd5a5d42f42aa9a69ec5b0ee2e8': {
```

Comment:
These are correct

Reply:
Why are they in bytes32 form? To somehow link with Govern?

Reply:
yes, and the same approach is used on Operate, so just for consistency. they are converted to addresses in create&stake:
> stakingInstance: getAddressFromBytes32(contract)

Reply:
ideally these would be address formats, but minor if already implemented to match with govern i guess